src/sections/upload-dataset-files/FileUploader.module.scss

```diff
@@ -47,3 +72,17 @@
     font-size: 1.3em;
     text-align: center;
 }
+
+.forms {
+    padding: 0.5em;
+}
+
+.selected_files_cb {
+    display: inline-flex;
+    align-items: center;
+}
+
+.save_btn {
+    float: right;
+    padding-top: 1rem;
+}
```

Comment:
Please add an empty line at the bottom. Prettier is not auto formatting this file for you? The indent is different on my vs code editor when I save this file.